src/modules/documentExecution/dashboard/generalSettings/background/DashboardBackground.vue

```diff
@@ -1,7 +1,7 @@
 <template>
     <div class="p-d-flex p-flex-column kn-flex p-mr-3 p-my-3 dashboard-card-shadow kn-overflow dashboard-scrollbar">
         <label class="kn-material-input-label p-m-3"> {{ $t('dashboard.widgetEditor.background') }}</label>
-
+        {{ background }}
```

Comment:
is this print (and the console log at line 56) intentional for debugging purpose or can we remove it?

Reply:
That one escaped me, removed!

Reply:
thank you!